tracker-radar-tests/TR-domain-matching/domain_matching_tests.json

```diff
@@ -117,6 +117,27 @@
                     "ios-browser"
                 ]
             },
+            {
+                "name": "Tracker URL, same entity; CNAME is tracker (ignore): ignore",
+                "siteURL": "https://third-party.site/",
+                "requestURL": "https://fake-ignore.tracker.test/spy/script.js",
+                "requestType": "script",
+                "expectAction": "ignore",
+                "exceptPlatforms": [
+                    "ios-browser"
+                ]
+            },
+            {
+                "name": "Non tracker URL; CNAME is tracker (block), same entity: ignore",
+                "siteURL": "https://third-party.site/",
+                "requestURL": "https://bad.cnames.test/spy/script.js",
+                "requestType": "script",
+                "expectAction": "ignore",
+                "exceptPlatforms": [
+                    "ios-browser",
+                    "android-browser"
```

Comment:
Can we please link up all those exceptions in the testset readme to asana tasks to address them?